pkg/tnf/test.go

```diff
@@ -33,6 +34,19 @@ const (
 	FAILURE
 )
 
+// TestsExtraInfo a collection of messages per test that is added to the claim file
+// use WriteTestExtraInfo for writing to it
+var TestsExtraInfo []map[string][]string = []map[string][]string{}
+
+// WriteTestExtraInfo adds a new object for the current test with the given info
+func WriteTestExtraInfo(info []string) {
```

Comment:
Please make this method able to add to an already existing entry in `TestsExtraInfo` for a test. As it is I think it's prone to unintentional misuse.

If `WriteTestExtraInfo` is called after any assertions have been made in a test it may never be called, and any information accumulated in the test up to that point (`extraInfo` in the example) will not be recorded.